.jenkins/actions/run_cached_fv_dynamics.sh

```diff
@@ -8,7 +8,10 @@ export TEST_ARGS="-v -s -rsx --backend=${BACKEND} --which_modules=FVDynamics"
 make get_test_data
 
 if [ ! -d $(pwd)/.gt_cache_000000 ]; then
-    cp -r /scratch/snx3000/olifu/jenkins/scratch/store_gt_caches/$EXPNAME/$BACKEND/.gt_cache_0000* .
-    find . -name m_\*.py -exec sed -i "s|\/scratch\/snx3000\/olifu\/jenkins_submit\/workspace\/fv3core-cache-setup\/backend\/$BACKEND\/experiment\/$EXPNAME\/slave\/daint_submit|$(pwd)|g" {} +
+    version=`cat /scratch/snx3000/olifu/jenkins/scratch/store_gt_caches/$EXPNAME/$BACKEND/version.txt`
+    if [ "$version" = "$GT4PY_VERSION" ]; then
```

Comment:
From where is `$GT4PY_VERSION` set here?

Reply:
from here:
https://github.com/VulcanClimateModeling/fv3core/blob/415cfb9356a6c9cea697c6cea70989865eb2ca3b/.jenkins/jenkins.sh#L131

Reply:
Maybe comment it?